### ISSUE TITLE
Pull after merging the release branch

### ISSUE THREAD

**Issue by jan-auer** [getsentry/craft#115]
After successfully publishing and merging the release branch, you still have to manually call `git pull` to get your local branch up to speed. Craft could do that automatically.

The only place where this probably shouldn't happen, is the GitHub Action.

**Comment by BYK** [comment]
I don't think this applies as Craft seems to be using the [merge API](https://developer.github.com/v3/repos/merging/). What we kept seeing it the protected branch stuff I think.

**Comment by jan-auer** [comment]
Not sure if I follow. Let me clarify:

When I use `craft publish` locally to release a new version of an SDK or library, my `master` branch is not pulled automatically. It happened to me a few times that I continued working after this and only noticed much later that I had to `git pull` to get the merge commit of the release branch in.

Since craft writes `Merging the release branch`, I subconsciously expect that afterwards my local checkout is up to date. That is why it would be a nice if craft pulls the local checkout after merging the release branch. (Obviously it should only do that when the checkout is on master and it is up-to-date pre-merge). 

This is independent of how craft uses the merge API, and I'm not sure what protected branches have to do with this.
